blazor/appearance/theme-studio.md

```diff
@@ -15,7 +15,7 @@ Theme Studio for Syncfusion Blazor can be used to customize a new theme from an
 
 The Syncfusion Blazor themes are developed under the SCSS environment. Each theme has a unique common variable list. When you change the common variable color code value, it will reflect in all the Syncfusion Blazor components. All Syncfusion Blazor component styles are derived from these [theme-based common variables](#common-variables). This common variable list is handled inside the Theme Studio application for customizing theme-based colors.
 
-1. Open [Blazor Theme Studio](https://blazor.syncfusion.com/themestudio/) application.
+1. Open [Blazor Theme Studio](https://ej2.syncfusion.com/themestudio/?theme=bootstrap5) application.
```

Comment:
Seems its not a blazor domain link. recheck it.

Reply:
Added blazor theme studio link